guides/dai_transaction_from_browser.md

```diff
@@ -0,0 +1,65 @@
+# Transacting DAI on OmiseGO Network through the browser
+
```

Comment:
Transacting with DAI on the OmiseGO Network (is 'through a browser' necessary?) Could it be:
* 'Using a browser for DAI transactions on the OmiseGO Network'
* Transacting DAI online through the OmiseGO Network